### DISCUSSION TITLE
help with gempy_viewer

### DISCUSSION THREAD

**Discussion by ljapril** [gempy-project/gempy#914] [Q&A]
Hello,
I downloaded the Greenstone jupyter notebook and was trying to run it but I am getting a Module Not Found Error:
<img width="887" alt="Screenshot 2024-06-25 at 12 58 40 PM" src="https://github.com/gempy-project/gempy/assets/126612508/baa34fb4-48f7-4f8e-879d-320bd449921c">
Could someone explain why this is happening?
Thank you.

**Comment by javoha** [comment]
Hi,
how did you install gempy? It looks like you just did not install gempy_viewer. I would recommend to use:
`pip install gempy[opt]`
That makes sure you have the crucial packages, namely gempy, gempy_viewer and gempy_engine.
Cheers
Jan
